gtsam/geometry/Cal3Bundler.cpp

```diff
@@ -118,6 +120,25 @@ Point2 Cal3Bundler::calibrate(const Point2& pi, const double tol) const {
     throw std::runtime_error(
         "Cal3Bundler::calibrate fails to converge. need a better initialization");
 
+  const double x = invKPi.x(), y = invKPi.y();
```

Comment:
remove invKPi variable, just init x and y on line 102, and use to init `pn` on line 105.

Reply:
We still need invKPi in the iteration...